modules/ROOT/pages/to-find-info.adoc

```diff
@@ -30,54 +13,141 @@ Exchange Search looks for terms in:
 * The content of the asset portal pages
 * xref:to-find-info.adoc#searching-api-specifications[Properties defined in RAML and OAS API specifications]
 
-You can combine search terms with the filters listed in All Types.
+On an asset portal, click a tag or category element to create a filtered search showing similar assets.
+
+The original Exchange search experience is available in Anypoint Platform Private Cloud Edition, MuleSoft Government Cloud, the US cloud, and the EU cloud. The new Exchange search experience is available in the US cloud and the EU cloud.
+
+New Exchange search experience:
+
+image::search.png[New Exchange search experience]
+
+== Tags, Categories, and Ranking in the New Exchange Search Experience
+
+To filter search results by tags and categories in the new Exchange search experience, use the drop-down menus for tags and categories on the search page.
 
-You can also search by clicking Category or Tag elements. This creates a filtered search you can use to find similar assets.
+To select the ranking of search results in the new Exchange search experience, use the *Sort by* drop-down menu to choose *Relevance* (default), *Name*, *Last modified*, or *Rating*. *Relevance* sorts the matching results by where a match is found in the asset metadata, such as name, title, pages, and API specification metadata. API specification metadata includes endpoints and types.
 
-In Exchange Private Cloud Edition and GovCloud, searches apply only to the Anypoint Exchange organization you are viewing. If you search in a private Exchange organization, the search only lists that organization's assets and not MuleSoft's public assets.
+== Searching in Organizations
 
-In Exchange Cloud, you can select which of the organizations visible to you should be included in each search.
+In the original Exchange search experience, searches apply only to the Exchange organization you are viewing. If you search in a private Exchange organization, the search only lists that organization's assets and not MuleSoft's public assets.
+
+In the new Exchange search experience, you can select which of the organizations visible to you should be included in each search. By default, no organization is selected in the drop-down menu, and Exchange searches MuleSoft's public assets and the assets of the master organization and its business groups.
```

Comment:
So is the distinction really of one org at a time (orig) vs. selected orgs or MS+master+all orgs?

A graphic might be great here, comparing orig search and new search?

Reply:
Updated.